### DISCUSSION TITLE
Formatting of HTML Closing Tags

### DISCUSSION THREAD

**Discussion by KennedyTedesco** [Stillat/blade-parser-typescript#88] [Q&A] [answered]
Hi there! This project is awesome and really promising. Kudos!

Currently, the HTML code is being formatted in a way that results in this indentation:

```blade
<div
    class="flex px-4 sm:justify-end sm:px-6"
>
    <x-filament::button
        class="mb-4 mr-2 last:mr-0 md:mb-0"
        color="success"
        icon="heroicon-m-plus-circle"
        icon-position="before"
    >
        Button
    </x-filament::button>
</div>
```

I want to place the `>` symbol immediately following the last attribute on the same line. Is there any configuration I can apply to make the formatter produce this instead?

```blade
<div
    class="flex px-4 sm:justify-end sm:px-6">
    <x-filament::button
        class="mb-4 mr-2 last:mr-0 md:mb-0"
        color="success"
        icon="heroicon-m-plus-circle"
        icon-position="before">
        Button
    </x-filament::button>
</div>
```

**Comment by JohnathonKoster** [comment, marked as answer]
This can be accomplished by using Prettier's `bracketSameLine` option (https://prettier.io/docs/en/options.html#bracket-line).

Here is what that might look like (just a sample file):

```json
{
    "bracketSameLine": true,
    "plugins": [
        "prettier-plugin-blade",
        "prettier-plugin-tailwindcss"
    ],
    "overrides": [
        {
            "files": [
                "*.blade.php"
            ],
            "options": {
                "parser": "blade"
            }
        }
    ]
}
```

**Reply by KennedyTedesco** [reply]
Awesome! Thank you very much. It's my first time using Prettier, good to know there are plenty of options there to be used.
